File: backend/src/main/java/tko/model/dto/user/RegisterUsersDTO.java
package tko.model.dto.user;

import tko.utils.Gender;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDate;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class RegisterUsersDTO {
    private String name;
    private String login;
    private String password;
    private LocalDate birthDay;
    private Double weight;
    private Double height;
    private Gender gender;
}

File: android/app/src/main/java/tk/ssau/fitnesstko/model/dto/user/RegisterUsersDTO.kt
package tk.ssau.fitnesstko.model.dto.user

data class RegisterUsersDTO(
    val name: String,
    val login: String,
    val password: String,
    val birthDay: java.time.LocalDate,
    val weight: Double,
    val height: Double,
    val gender: Gender
)

enum class Gender { MALE, FEMALE }
